.lintstagedrc.json

```
@@ -1,6 +1,3 @@
 {
-  "*.{ts,tsx}": [
-    "yarn lint-code-style",
-    "git add"
-  ]
+  "*.{ts,tsx}": ["yarn lint-code-style", "yarn test", "git add"]
```

I would suggest to not run the tests when adding changes to stage. It can slow down the development process since every time that a commit is made, it will take more than 30 seconds in order to execute all these scripts. 

```suggestion
  "*.{ts,tsx}": ["yarn lint-code-style", "git add"]
```